types/facebook-nodejs-business-sdk/tslint.json

```diff
@@ -0,0 +1,12 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "ban-types": false,
```

Comment:
for new packages, please try to override rules with per-line tslint-ignores.

If this is copied from some other tslint.json, chances are good that you don't need any of these overrides.

Reply:
Rules overrides removed.